src/App.tsx

```diff
@@ -86,8 +89,10 @@ const App: React.FC = () => {
       >
         <RMWCThemeProvider className={`app-container ${theme.type}-theme`} options={theme}>
           <CssBaseline />
-          <StaticDataFetcher />
-          <DiscreteLayerView />
+          <EnumsMapContext.Provider value={{ enumsMap, setEnumsMap }}>
```

Comment:
setEnumsMap used from <MCEnums> which changes outer context value... less preferable...but it happens only once .

Reply:
Not very clear.. could you rephrase please?

Reply:
You change the context value from inner/nested component basically you should only read context value in nested components